.cirrus.yml

```diff
@@ -166,3 +166,41 @@ static_binary_task:
     save_cache_script: |
         mv /nix .cache
         chown -Rf $(whoami) .cache
+
+# Build the static binary for arm64
+static_binary_arm64_task:
+    depends_on:
+        - 'config'
+        - 'fmt'
+
+    gce_instance:
+        image_name: "${FEDORA_CACHE_IMAGE_NAME}"
+        cpu: 8
+        memory: 12
+        disk: 200
+
+    init_script: |
+        set -ex
+        setenforce 0
+        growpart /dev/sda 1 || true
+        resize2fs /dev/sda1 || true
+        yum -y install podman
+
+    nix_arm64_cache:
+      folder: '.cache'
+      fingerprint_script: |
+        echo "nix-v1-$(sha1sum nix/nixpkgs.json | head -c 40)"
+
+    build_script: |
+        set -ex
+        mkdir -p .cache
+        mv .cache /nix
+        if [[ -z $(ls -A /nix) ]]; then podman run --rm --privileged -ti -v /:/mnt nixos/nix cp -rfT /nix /mnt/nix; fi
+        podman run --rm --privileged -ti -v /nix:/nix -v ${PWD}:${PWD} -w ${PWD} nixos/nix nix --print-build-logs --option cores 8 --option max-jobs 8 build --file nix/default-arm64.nix
```

Comment:
It just occurred to me, this will hit the docker-hub, and so could fail randomly (do we do this in podman?).  Would it make sense to point at (or configure) `mirror.gcr.io` proxy for the image?

Reply:
Yes, I think this makes sense. Added it as suggested.

Reply:
I tried it but it looks like that the mirror does not cache the nix image: https://cirrus-ci.com/task/5561064597225472

Reply:
Hmmm, dang.  Somehow we've managed to not have problems (that I've observed) on podman, so it's probably okay for now.  Esp. on this low-traffic repo.